.github/workflows/ci.yml

```diff
@@ -53,20 +53,26 @@ jobs:
       fail-fast: true
       matrix:
         distro:
-          - ubuntu2004
-          - debian10
+          - "ghcr.io/tobiashuste/ubuntu-systemd:18.04"
+          - "ghcr.io/tobiashuste/ubuntu-systemd:20.04"
+          - "docker.io/geerlingguy/docker-debian10-ansible"
 
     steps:
       - name: Check out the codebase.
         uses: actions/checkout@v3
 
+      - name: Enable lingering required for podman and systemd in GH Actions.
+        run: |  # Compare with https://github.com/eriksjolund/user-systemd-service-actions-workflow/blob/efe872924fd2dd35bb482544126ce751303e14c2/README.md
+          sudo loginctl enable-linger $UID
+          sleep 1
+
       - name: Prepare the job environment.
         uses: ./.github/workflows/prepare-action
 
       - name: Run Molecule tests.
-        run: poetry run molecule test
+        run: XDG_RUNTIME_DIR=/run/user/$UID poetry run molecule test
         env:
-          MOLECULE_DISTRO: ${{ matrix.distro }}
+          MOLECULE_IMAGE: ${{ matrix.image }}
```

Comment:
I wonder why this works.. shouldn't that be `MOLECULE_IMAGE: ${{ matrix.distro }}` ?

Reply:
Probably it always falls back to the default and never uses different images.